.github/workflows/release.yml

```diff
@@ -55,14 +55,15 @@ jobs:
     needs: [release-please]
     if: ${{ needs.release-please.outputs.tag-name }}
     runs-on: ubuntu-latest
+    # TODO: Switch to use `workflow_call`?
     steps:
-      - name: Dispatch to build-nargo
+      - name: Dispatch to publish workflow
         uses: benc-uk/workflow-dispatch@v1
         with:
           workflow: publish.yml
-          repo: noir-lang/build-nargo
+          repo: noir-lang/noir
           ref: master
-          token: ${{ secrets.NOIR_REPO_TOKEN }}
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I'm not sure everywhere this token might be used, but if this was the only place, we'd want to purge it from the system

Reply:
`NOIR_REPO_TOKEN` you mean?

Reply:
Yeah, the more we are able to use `GITHUB_TOKEN`, the better. So we just need to remove any unused custom secrets. Probably need to grep other workflows for`NOIR_REPO_TOKEN`

Reply:
Yeah, we only use it for triggering the `noir_wasm` release workflow but that should be handled with https://github.com/noir-lang/noir/issues/2210

Reply:
Actually.... that shouldn't need this token at all.